### DISCUSSION TITLE
抢到票了感谢

### DISCUSSION THREAD

**Discussion by Kinoko7c7** [mikumifa/biliTickerBuy#317] [聊天专区]
![14cd0854a765365cb57468001673aa84](https://github.com/mikumifa/biliTickerBuy/assets/113074631/d57d657f-4587-4a33-867c-c3c96eb92154)
300ms，自动，1秒就抢到了

**Comment by Kinoko7c7** [comment]
开始抢票时间是windows时间，建议和手机上时间对一下，或者提前半分钟开始抢票

**Comment by Kinoko7c7** [comment]
很多人设置都和我一样，感觉就比较吃运气了

**Reply by JYK89** [reply]
验证你是手动还是自动验证的

**Reply by Kinoko7c7** [reply]
自动验证
